### ISSUE TITLE
`hosting:channel:create` is ignoring the given `channelId` in the latest release

### ISSUE THREAD

**Issue by Flambe** [firebase/firebase-tools#3417]
### [REQUIRED] Environment info

**firebase-tools:** 9.12.0

**Platform:** Ubuntu, Debian (using the latest docker image andreysenov/firebase-tools)

### [REQUIRED] Test case

1. Create a new directory and run `firebase init hosting` with the default options

### [REQUIRED] Steps to reproduce

2. In the new project, run `firebase hosting:channel:create some-channel-name`

### [REQUIRED] Expected behavior

* A new channel is created with the channel id `some-channel-name` (this is what happens on v9.11.0)

### [REQUIRED] Actual behavior

* I get prompted for the channel ID `? Please provide a URL-friendly name for the channel:`
`--debug`:
```bash
$ firebase hosting:channel:create some-channel-name --debug
[2021-05-27T08:51:51.573Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[2021-05-27T08:51:51.573Z] > authorizing via signed-in user (steve_axtmann@intergral.com)
[2021-05-27T08:51:51.574Z] [iam] checking project steve-nv-bot for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[2021-05-27T08:51:51.575Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[2021-05-27T08:51:51.576Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[2021-05-27T08:51:51.770Z] <<< HTTP RESPONSE 200 {"date":"Thu, 27 May 2021 08:51:51 GMT","pragma":"no-cache","cache-control":"no-cache, no-store, max-age=0, must-revalidate","expires":"Mon, 01 Jan 1990 00:00:00 GMT","content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","server":"scaffolding on HTTPServer2","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-29=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[2021-05-27T08:51:51.791Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/steve-nv-bot:testIamPermissions  
 {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[2021-05-27T08:51:52.008Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Thu, 27 May 2021 08:51:51 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","server-timing":"gfet4t7; dur=123","alt-svc":"h3-29=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[2021-05-27T08:51:52.012Z] > refreshing access token with scopes: []
[2021-05-27T08:51:52.012Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[2021-05-27T08:51:52.147Z] <<< HTTP RESPONSE 200 {"expires":"Mon, 01 Jan 1990 00:00:00 GMT","date":"Thu, 27 May 2021 08:51:52 GMT","pragma":"no-cache","cache-control":"no-cache, no-store, max-age=0, must-revalidate","content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","server":"scaffolding on HTTPServer2","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-29=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[2021-05-27T08:51:52.166Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/steve-nv-bot [none]
[2021-05-27T08:51:52.456Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects/steve-nv-bot 200
[2021-05-27T08:51:52.457Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/steve-nv-bot {"projectId":"steve-nv-bot","projectNumber":"121152045006","displayName":"steve-nv-bot","name":"projects/steve-nv-bot","resources":{"hostingSite":"steve-nv-bot","storageBucket":"steve-nv-bot.appspot.com","locationId":"europe-west3"},"state":"ACTIVE"}
? Please provide a URL-friendly name for the channel: 
```

**Comment by service-paradis** [comment]
I had the same issue with `firebase-tools@9.12.0`. Reverting to `firebase-tools@9.11.0` since it broke our CI.

**Comment by bkendall** [comment]
This is fixed in `v9.12.1`
